pkgs/unified_analytics/lib/src/constants.dart

```diff
@@ -88,7 +88,16 @@ platform, Flutter framework, and related tools.
 Telemetry is not sent on the very first run.
```

Comment:
My understanding is that if the user does not disable analytics after this message is shown, telemetry will be collected on subsequent runs. Can we make that clear - `Telemetry is not sent on the very first run` , but will be sent on subsequent runs. To stop collection of telemetry, make sure to run the disable command etc. Some more messaging to make this clear.

Reply:
How does this sound?

```
Flutter and Dart related tooling uses Google Analytics to report usage and
diagnostic data along with package dependencies, and crash reporting to
send basic crash reports. This data is used to help improve the Dart
platform, Flutter framework, and related tools.

Telemetry is not sent on the very first run, but will be sent
on subsequent runs if not disabled explicitly.

To disable reporting of telemetry, run this terminal command:

flutter config --no-analytics
  OR
dart --disable-analytics

To enable reporting of telemetry, run this terminal command:

flutter config --analytics
  OR
dart --enable-analytics

You can use either the flutter or dart command if both are installed,
they will set the telemetry status for both tools and any other
related tooling.

If you opt out of telemetry, an opt-out event will be sent, and then no further
information will be sent. This data is collected in accordance with the
Google Privacy Policy (https://policies.google.com/privacy).
```

Reply:
LGTM!